modules/apps/blogs/blogs-service/src/main/java/com/liferay/blogs/model/impl/BlogsEntryImpl.java

```diff
@@ -126,6 +140,21 @@ public void setSmallImageType(String smallImageType) {
 		_smallImageType = smallImageType;
 	}
 
+	private FileEntry _getFileEntry(long fileEntryId) {
```

Comment:
Can you rename this method to `_fetchFileEntry`? in general, `get*` methods should return a value or throw `NoSuch*` if there's none available.